src/components/Content.tsx

```diff
@@ -120,9 +119,7 @@ const Content: ContentComponent = props => {
     if (focusNode && focusNode.nodeType === Node.TEXT_NODE) return
 
     // if disableOnFocus is true, the click came from an Editable onFocus event and we should not reset the cursor
-    if (showModal) {
-      dispatch(modalRemindMeLater({ duration: MODAL_CLOSE_DURATION }))
-    } else if (!noteFocus) {
+    if (!showModal && !noteFocus) {
```

Comment:
I assume this should be replaced with `closeModal` rather than removing?

Reply:
But this is just passing the duration which was used to set `hideUntil`. And since we aren't using `hideUntil` anymore, I thought we should be ok to delete this. I will take a look again at this.

Reply:
`modalRemindMeLater` closed the current Modal and set `hideuntil`.  We still need to close the Modal.